### DISCUSSION TITLE
How to use Confident Learning to blindly believe that a model prediction is correct and no need to verify again about whether that prediction is correct or not

### DISCUSSION THREAD

**Discussion by Giriteja** [cleanlab/cleanlab#602] [Q&A] [answered]
How to use Confident Learning to blindly believe that a model prediction is correct and no need to verify again about whether that prediction is correct or not by manually checking

**Comment by jwmueller** [comment, marked as answer]
Sorry we missed your question, but have an answer this and other common questions in our FAQ:  
https://docs.cleanlab.ai/master/tutorials/faq.html#How-do-I-infer-the-correct-labels-for-examples-cleanlab-has-flagged?

TLDR: is you should not blindly believe anything!
